sandstorm-pkgdef.capnp

```diff
@@ -184,6 +224,7 @@ const myCommand :Spk.Manifest.Command = (
   environ = [
     # Note that this defines the *entire* environment seen by your app.
     (key = "PATH", value = "/usr/local/bin:/usr/bin:/bin"),
+    (key = "SANDSTORM", value = "1"),
```

Comment:
If you are not compiling for Sandstorm, the sandstorm-pkgdef.capnp file is never used. But, if you are compiling a Sandstorm SPK file, this setting in the package definition file will set up the environment variable SANDSTORM=1, which can be used elsewhere in the app to determine (if present) that the app should do something different for Sandstorm users.

So, for everyone else, this variable never appears, and so Sandstorm-specific code that checks for it won't run.